docs/inventory-plugins.rst

```diff
@@ -1,229 +1,117 @@
+  - **Project**: Required. Pre-populates with a default project, otherwise, specify the project this inventory is using as its source. Click the |search| button to choose from a list of projects. If the list is extensive, use the search to narrow the options.
+  - **Inventory File**: Required. Select an inventory file associated with the sourced project. If not already populated, you can type it into the text field within the drop down menu to filter the extraneous file types. In addition to a flat file inventory, you can point to a directory or an inventory script.
 
-2. The Create Source window expands with the required **Credential** field. Choose from an existing Red Hat Virtualization Credential.
+.. |search| image:: _static/images/search-button.png
 
-3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as needed.
 
-4. Use the **Source Variables** field to override variables used by the ``ovirt`` inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For a detailed description of these variables, view the `ovirt inventory plugin <https://cloud.redhat.com/ansible/automation-hub/repo/published/redhat/rhv/content/inventory/ovirt>`__.
+3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as described in the main procedure for :ref:`adding a source <ug_add_inv_common_fields>`.
 
-|Inventories - create source - RHV example|
+4. In order to pass to the custom inventory script, you can optionally set environment variables in the **Environment Variables** field. You may also place inventory scripts in source control and then run it from a project. See :ref:`ag_inv_import` for detail.
 
-.. |Inventories - create source - RHV example| image:: _static/images/inventories-create-source-rhv-example.png
-   :alt: Inventories create source Red Hat Virtualization example
+|Inventories - create source - sourced from project example|
 
+.. |Inventories - create source - sourced from project example| image:: _static/images/inventories-create-source-sourced-from-project-example.png
 
-.. note::
+.. note:: If you are executing a custom inventory script from SCM, please make sure you set the execution bit (i.e. ``chmod +x``) on the script in your upstream source control. If you do not, AWX will throw a ``[Errno 13] Permission denied`` error upon execution.
 
-  Red Hat Virtualization (ovirt) inventory source requests are secure by default. To change this default setting, set the key ``ovirt_insecure`` to **true** in ``source_variables``, which is only available from the API details of the inventory source at the ``/api/v2/inventory_sources/N/`` endpoint.
 
 .. _ug_source_rhaap:
 
 Red Hat Ansible Automation Platform
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. index::
-   pair: inventories; Red Hat Ansible Automation Platform
-
+-----------------------------------
```

Comment:
Looks like this could be separate.

Reply:
All tests are passing now; I am not going to split it out, except in another commit.